### PR TITLE
gophercloud cdn services getting started samples

### PR DESCRIPTION
**Pull request by jrperritt**

### CODE REVIEW

**Comment by ycombinator** [comment]
@smashwilson, @jamiehannaford I'm not familiar with Go and gophercloud; would one of you mind reviewing this PR and merging? Thanks.

**Comment by smashwilson** [comment]
Reviewing now.

File: src/docs/cdn/getting-started/samples/create_service.rst [hunk 1]
```diff
@@ -4,7 +4,21 @@
 
 .. code-block:: go
 
-  // Not currently supported by this SDK
+  createOpts := osServices.CreateOpts{
```
Comment (smashwilson) [inline src/docs/cdn/getting-started/samples/create_service.rst]: There's no OpenStack import up above for `osServices` :point_up:

**Comment by smashwilson** [comment]
Other than the missing imports, these are fine :smile:

**Comment by jrperritt** [comment]
I was going off of the other Getting Started samples. For example, here: https://github.com/rackerlabs/developer.rackspace.com/blob/master/src/docs/cloud-servers/getting-started/samples/upload_existing_keypair.rst and here: https://github.com/rackerlabs/developer.rackspace.com/blob/master/src/docs/cloud-block-storage/getting-started/samples/create_volume.rst 
I haven't written these before; it looks like if we want to import the delegating packages, some of these sample files will need to be changed.

**Comment by smashwilson** [comment]
@jrperritt: check the "authentication" samples instead:

https://github.com/rackerlabs/developer.rackspace.com/blob/master/src/docs/cloud-servers/getting-started/samples/authentication.rst
https://github.com/rackerlabs/developer.rackspace.com/blob/master/src/docs/cloud-block-storage/getting-started/samples/authentication.rst

I'd only been putting them in the first sample on each page, but I'd been trying to import everything that we use in the guide there.

If we *are* missing an "os" import that we use, yeah, I'd want to fix that... otherwise it's pretty confusing for a user who's not acquainted with the way our SDK is structured and delegation is done.

**Comment by jrperritt** [comment]
OK, cool, I'll add them in the `authentication` sample.

**Comment by jrperritt** [comment]
OK, I added the OpenStack imports. In addition, the original Rackspace import paths were wrong... and those are fixed, too.

**Comment by smashwilson** [comment]
:bow: Awesome, looks great. Merging :zap:

**Comment by jrperritt** [comment]
Thank you :)

**Pull request merged**